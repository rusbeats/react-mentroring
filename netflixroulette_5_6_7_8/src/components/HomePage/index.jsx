import React, { Fragment } from 'react';
import Header from '../Header';
import Films from '../FilmsList/Films';
import Footer from '../Footer';

//Wrapped in Fradment
//Removed all unnecessary divs
//included Header and Footer
const HomePage = () => {
    return (
        <Fragment>
            <Header />
            <Films />
            <Footer />
        </Fragment>
    );
};

export default HomePage;
