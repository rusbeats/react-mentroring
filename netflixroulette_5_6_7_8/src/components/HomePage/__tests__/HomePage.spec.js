import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../index';

describe('HomePage', () => {
    it('should render correctly (snapshot)', () => {
        const component = shallow(<HomePage />);
        expect(component).toMatchSnapshot();
    });
});
