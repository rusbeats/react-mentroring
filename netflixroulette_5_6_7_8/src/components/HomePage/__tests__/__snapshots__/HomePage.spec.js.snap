// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render correctly (snapshot) 1`] = `
<Fragment>
  <Connect(Header) />
  <Connect(Films) />
  <Footer />
</Fragment>
`;
